RELEASE_NOTES.md

```diff
@@ -0,0 +1,79 @@
+# Release v0.10.1 #
+
+## New Features ##
+
+* Admin can configure **MaxCapacity** through REST API for a given Virtual Cluster so that the virtual cluster can use iddle resources as bonus. [#2147](https://github.com/Microsoft/pai/pull/2147)
+* Support Azure RDMA [#2091](https://github.com/Microsoft/pai/pull/2091); [how-to doc](https://github.com/Microsoft/pai/blob/master/docs/pai-management/doc/azure/enable-az-rdma.md) 
+* New disk cleaning support to mitigate the disk pressure by killing job container that consumes most of the disk. [#2119](https://github.com/Microsoft/pai/pull/2119)
+* Web portal: add "My jobs" filter button. [#2111](https://github.com/Microsoft/pai/pull/2111)
+* "Submit Simple Job" web portal plugin [#2131](https://github.com/Microsoft/pai/pull/2131)
+
+## Improvements ##
+
+### Service ###
+
+* Hadoop: disable hdfs shortcircuit [#2027](https://github.com/Microsoft/pai/pull/2027)
+* Hadoop: Expose log retain time [#2034](https://github.com/Microsoft/pai/pull/2034)
+* Hadoop: Reduce am resource [#2072](https://github.com/Microsoft/pai/pull/2072)
+* Kubernetes: Disable kubernetes's pod eviction [#2124](https://github.com/Microsoft/pai/pull/2124)
+* Pylon: WebHDFS library compatibility [#2134](https://github.com/Microsoft/pai/pull/2134)
+* Hadoop: Extend nm expiry time to 60 mins [#2142](https://github.com/Microsoft/pai/pull/2142)
```

Comment:
Extend the NM expiry time from 15 mins to 60 mins to provide a better tolerable experience for NM downtime.